scripts/check-form.js

```diff
@@ -19,16 +19,25 @@ if (!key) {
 
 const sheetData = `https://sheets.googleapis.com/v4/spreadsheets/${sheetID}/values/Sheet1!A2:A?key=${key}`;
 
-const [,, slug, branch, all] = process.argv;
-
-if (!slug || !branch) {
-	throw 'args required: slug, branch';
+const { values, positionals } = require('util').parseArgs({
+  options: { all: { type: 'boolean', default: false } },
+  allowPositionals: true,
+  allowNegative: true,
+  strict: true,
+});
+if (positionals.length < 1 || positionals.length > 2) {
+	throw 'usage: node check-form.js <slug> [--all] [ref]';
 }
-if (typeof all !== 'undefined' && all !== '--all') {
-	throw '`all` arg, if provided, must be `--all`'
+const [slug, ref = 'HEAD'] = positionals;
+const { all } = values;
```

Comment:
wait also is `all` even being used?

Reply:
yes, `npm run ipr-check` passes it.

Reply:
No I mean in the script.

Reply:
ah, you're right, i refactored out the usage but not the flag. i'll remove that.